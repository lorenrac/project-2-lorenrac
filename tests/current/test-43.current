When will it end?
ERROR (line 2): Unexpected End-of-File
