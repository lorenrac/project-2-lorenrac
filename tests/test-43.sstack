// ERROR: Missing close brace
IF ("1") {
  PRINT "When will it end?"
