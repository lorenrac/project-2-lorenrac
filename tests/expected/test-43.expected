ERROR (line 4): Unexpected End-of-File
